contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -315,6 +316,7 @@ func (s *SchedulerServer) addCoreFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.minionLogMaxAgeInDays, "minion-max-log-age", s.minionLogMaxAgeInDays, "Maximum log file age of the executor and proxy in days")
 	fs.IntVar(&s.minionLogMaxBackups, "minion-max-log-backups", s.minionLogMaxBackups, "Maximum log file backups of the executor and proxy to keep after rotation")
 
+	fs.StringSliceVar(&s.kubeletApiServerList, "kubelet-api-servers", s.kubeletApiServerList, "List of Kubernetes API servers for publishing events, and reading pods and services. (ip:port), comma separated. To be used by kubelet")
```

Comment:
please indicate that it's optional and that if unspecified it defaults to the value of `api-servers`